index.html

```diff
@@ -8,7 +8,7 @@
        <title>My Mobile Portfolio</title>
     </head>
     <body>
-        <header>
+        <header class ="me">
```

Comment:
Your layout is not matching the Figma template of your choosing. Kindly change and ask for a re-review.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178109396-933b0aa5-5b22-4a6f-85cd-5b712c224334.png) | ![image](https://user-images.githubusercontent.com/33268927/178109410-89d674be-4233-4c2b-9b64-476b1bff7536.png) |